modules/database/src/adapters/sequelize-adapter/SequelizeSchema.ts

```diff
@@ -134,7 +134,14 @@ export class SequelizeSchema extends SchemaAdapter<ModelStatic<any>> {
       false,
       options?.userId,
       options?.scope,
-    ).then(r => r!._id);
+    ).then(r => {
+      if (!r) {
```

Comment:
Should return 404 simply, not found. The message says more than it needs, and returning the id isn't correct